src/Validation/URLValidationRESTController.php

```diff
@@ -161,6 +216,24 @@ public function create_item_permissions_check( $request ) { // phpcs:ignore Vari
 		return true;
 	}
 
+	/**
+	 * Checks whether the current user can view results of a URL AMP validation.
+	 *
+	 * @param  WP_REST_Request $request Full details about the request.
+	 * @return true|WP_Error True if the request has permission; WP_Error object otherwise.
+	 */
+	public function get_item_permissions_check( $request ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

Comment:
I was thinking if we could reuse and rename the `create_item_permissions_check()` method instead of adding a new checker just for the `READABLE` method. Right now, they are the same.